syft/tensor.py

```diff
@@ -1397,6 +1399,35 @@ def mm(self, tensor2):
 
         return syft.mm(self, tensor2)
 
+    def stride(self, dim=None):
+        """
+        Returns the jump necessary to go from one element to the next one in the specified dimension dim.
+
+        :param dim: dimension
```

Comment:
This still hasn't resonated with the expected style of documentation.
use Parameters and Results and seperators
https://github.com/OpenMined/PySyft/blob/4e2aa9e34df985f4d8e0703d900eaa2558fd413a/syft/math.py#L259